src/locations/model/LocationClosedDay.ts

```diff
@@ -0,0 +1,6 @@
+export default interface LocationClosedDay {
+  createdAt: Date;
+  date: string;
```

Comment:
Any reason we can't use date object here @cduda @lmkerr ? I know it's not a full iso8601 but should be enough to have as the date type, right?

Reply:
@ryankazokas  @lmkerr timezone considerations was my thought. Where we are using this, I need to know what timezone the date would be in. I don't see anywhere where that is specified.

Reply:
^ This is a good point 
@ryankazokas What timezone are the location SLA times based off of?  The business timezone?

Reply:
@mb-ca @ryankazokas I really do think we need to add a timezone field to the location. Might need to add a timezone field to the location interface. I don't know if this changes anything, but I have already written code to convert the "date" to the envar passed to the step that determines timezone. As long as we setting on a standard where that envar is present when running a step we will not need to make any DB changes or API changes.

Reply:
@mb-ca @cduda , The core API is just handling them as strings right now as whatever we get them in. I think we chatted with Product about this, and the consensus was to just to make as simple as possible, hence the implementation. I agree. having a timezone in the SLA and actually treating them as full dates is probably a better experience here. For now, I think the answer would be to just save the string as the date in the business timezone so that it's consistent. Also, this will allow us to actually change this in the future.

Sorry I forgot about this. I'm good with leaving as a date for now, and I"ll throw something to get this handled in future.